kivymd/uix/snackbar.py

```diff
@@ -309,7 +316,9 @@ class BaseSnackbar(MDCard):
     All Snackbars will be made off of this `BaseSnackbar`. `BaseSnackbar` will always try to fill the remainder of the screen with your Snackbar.
     To make your Snackbar dynamic and symetric with snackbar_x. Set size_hint_x like below:
 
-    `size_hint_z = (Window.width - (snackbar_x * 2)) / Window.width`
+    `size_hint_z = (
+        Window.width - (snackbar_x * 2)
+    ) / Window.width`
 
     :Events:
         :attr:`on_open`
```

Comment:
It is better to place information about available events at the very top of the documentation line - https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/backdrop.py#L207

Reply:
Fixed.